ntbs-service/Services/AuthorizationService.cs

```diff
@@ -10,7 +10,7 @@ namespace ntbs_service.Services
 {
     public interface IAuthorizationService
     {
-        Task<PermissionLevel> GetPermissionLevelForNotificationAsync(ClaimsPrincipal user, Notification notification, NotificationGroup group);
+        Task<PermissionLevel> GetPermissionLevelForNotificationAsync(ClaimsPrincipal user, Notification notification, NotificationGroup group = null);

```

Comment:
`Notification` has `NotificationGroup` as a property - could we just use that, rather than passing through a new argument?